po/pl.po

```diff
@@ -8,201 +8,205 @@ msgstr ""
 
 #: data/resources/ui/login.ui:111
 msgid "This requires a restart to apply"
-msgstr ""
+msgstr "Wymagane ponowne uruchomienie, aby zastosować"
 
 #: data/resources/ui/login.ui:149
 msgid "Enter the Verification Code"
-msgstr ""
+msgstr "Wpisz Kod Weryfikacyjny"
 
 #: data/resources/ui/login.ui:168
 msgid "Code"
-msgstr ""
+msgstr "Kod"
 
 #: data/resources/ui/login.ui:205
 msgid "Register New Account"
-msgstr ""
+msgstr "Zarejestruj Nowe Konto"
 
 #: data/resources/ui/login.ui:225
 msgid "First Name"
-msgstr ""
+msgstr "Imię"
 
 #: data/resources/ui/login.ui:242
 msgid "Last Name"
-msgstr ""
+msgstr "Nazwisko"
 
 #: data/resources/ui/login.ui:205
 msgid "Enter Your Password"
-msgstr ""
+msgstr "Wpisz Swoje Hasło"
 
 #: data/resources/ui/login.ui:224
 msgid "Password"
-msgstr ""
+msgstr "Hasło"
 
 #: data/resources/ui/login.ui:261
 msgid "Enter the Encryption Key"
-msgstr ""
+msgstr "Wpisz Klucz Szyfrujący"
 
 #: data/resources/ui/shortcuts.ui:11
 msgctxt "shortcut window"
 msgid "General"
-msgstr ""
+msgstr "Ogólne"
 
 #: data/resources/ui/shortcuts.ui:14
 msgctxt "shortcut window"
 msgid "Show Shortcuts"
-msgstr ""
+msgstr "Pokaż Skróty Klawiszowe"
 
 #: data/resources/ui/shortcuts.ui:20
 msgctxt "shortcut window"
 msgid "Quit"
-msgstr ""
+msgstr "Wyjdź"
 
 #: data/resources/ui/sidebar.ui:6
 msgid "_Log Out"
-msgstr ""
+msgstr "_Wyloguj Się"
 
 #: data/resources/ui/sidebar.ui:12
 msgid "_Keyboard Shortcuts"
-msgstr ""
+msgstr "_Skróty Klawiszowe"
 
 #: data/resources/ui/sidebar.ui:16
 msgid "_About Telegrand"
-msgstr ""
+msgstr "_O Telegrandzie"
 
 #: src/window.rs:45 src/session/sidebar/chat_row.rs:240
 msgid "Photo"
-msgstr ""
+msgstr "Zdjęcie"
 
 #: src/window.rs:64 src/session/sidebar/chat_row.rs:259
 msgid "GIF"
-msgstr ""
+msgstr "GIF"
 
 #: src/window.rs:73 src/session/sidebar/chat_row.rs:268
 msgid "Video"
-msgstr ""
+msgstr "Video"
 
 #: src/window.rs::91 src/session/sidebar/chat_row.rs:286
 msgid "Voice message"
-msgstr ""
+msgstr "Wiadomość głosowa"
 
 #. Translators: '{}' is the placeholder for the full name of the contact who joined Telegram
 #: src/window.rs::110 src/session/sidebar/chat_row.rs:311
 msgid "{} joined Telegram"
-msgstr ""
+msgstr "{} dołączył do Telegrama"
 
 #: src/window.rs:112 src/session/sidebar/chat_row.rs:313
 msgid "Unsupported message"
-msgstr ""
+msgstr "Nieznana wiadomość"
 
 #: src/window.rs:409
 msgid "Incoming call"
-msgstr ""
+msgstr "Przychodzące połączenie"
 
 #. Translators: This is a date format in the day divider without the year
 #: src/session/content/item_row.rs:104
 msgid "%B %e"
-msgstr ""
+msgstr "%B %e"
 
 #. Translators: This is a date format in the day divider with the year
 #: src/session/content/item_row.rs:107
 msgid "%B %e, %Y"
-msgstr ""
+msgstr "%B %e %Y"
 
 #: src/session/content/message_row.rs:95
 msgid "This message is unsupported"
-msgstr ""
+msgstr "Ta wiadomość jest nieznana"
 
 #: src/session/sidebar/chat_row.rs:318
 msgid "You"
-msgstr ""
+msgstr " "
```

Comment:
Is the whitespace here intentional?

Reply:
Well, I don't think the author will ever get notified about this, so I guess the best thing to do here is deleting the string completely so it will appear as untranslated by weblate.

Reply:
oh sorry I misunderstood the context of this string, you're right it should be "Ty"